source/s3_default_meta_request.c

```diff
@@ -216,14 +212,6 @@ static bool s_s3_meta_request_default_update(
     return work_remaining;
 }
 
-static bool s_is_get_request(const struct aws_http_message *message) {
```

Comment:
i've seen similar logic for put somewhere as well. we should probably get rid of that too

Reply:
yeah, I removed [here](https://github.com/awslabs/aws-c-s3/pull/219/files#diff-436560b33218855638d58d1c14ecca658afd2c8ec65f2b525e21c6d829a7068aL658)